src/modules/Builder/pages/BuilderApplet/BuilderApplet.schema.ts

```diff
@@ -611,7 +611,8 @@ const SubscaleTableDataItemSchema = (featureFlags: FeatureFlags) =>
             severity: yup
               .string()
               .nullable()
-              .matches(createRegexFromList(Array.from(TScoreSeverity))),
+              // This includes the empty string so that there can be rows without a severity value
```

Comment:
I dont have problem with comments, but how does it works at this project ?
should we wipe or left comments ?

Reply:
I don't know that we have an agreed upon comment policy, but I try to leave in comments that can save a developer having to unnecessarily investigate something or ask a question. In this case, the empty string is not a valid value for severity, so I thought it worth it to explain why it's here.

Let me know how you feel about keeping this in